kamu-cli/src/commands/pull_command.rs

```diff
@@ -355,30 +344,15 @@ impl Command for PullCommand {
 struct PrettyPullProgress {
     multi_progress: Arc<indicatif::MultiProgress>,
     fetch_uncacheable: bool,
-    finished: Arc<AtomicBool>,
 }
 
 impl PrettyPullProgress {
     fn new(fetch_uncacheable: bool) -> Self {
         Self {
             multi_progress: Arc::new(indicatif::MultiProgress::new()),
             fetch_uncacheable,
-            finished: Arc::new(AtomicBool::new(false)),
         }
     }
-
-    fn draw(&self) {
-        loop {
-            if self.finished.load(Ordering::SeqCst) {
-                break;
-            }
-            std::thread::sleep(std::time::Duration::from_millis(100));
-        }
-    }
-
-    fn finish(&self) {
-        self.finished.store(true, Ordering::SeqCst);
-    }
```

Comment:
Looks like after some major upgrade of `indicatif` this code became meaningless and actually didn't refresh the progress bars. This is why I noticed that our UI became less "responsive". 

I replaced this with `enable_steady_tick()` calls.